### PR TITLE
Wazo 2909 python3

### PR DESCRIPTION
**Pull request by sopelj**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -4,7 +4,8 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, pycodestyle, pylint
+basepython = python3.7
+envlist = poy37, black, flake8, linters
```
Comment (bloom1) [inline tox.ini]: `poy37` is surely a typo :smile_cat:

Comment (bloom1) [reply tox.ini]: Fixed :+1:

**Review by bloom1 — CHANGES_REQUESTED** [review]
I have an issue with the argument parser when trying `wazo-provd-cli --help`: 
```shell
Traceback (most recent call last):
  File "/usr/bin/wazo-provd-cli", line 3, in <module>
    from wazo_provd_cli import cli
  File "/usr/lib/python3/dist-packages/wazo_provd_cli/cli.py", line 56, in <module>
    opts, args = parser.parse_known_args()
  File "/usr/lib/python3.7/argparse.py", line 1797, in parse_known_args
    self.error(str(err))
  File "/usr/lib/python3.7/argparse.py", line 2508, in error
    self.print_usage(_sys.stderr)
  File "/usr/lib/python3.7/argparse.py", line 2478, in print_usage
    self._print_message(self.format_usage(), file)
  File "/usr/lib/python3.7/argparse.py", line 2444, in format_usage
    return formatter.format_help()
  File "/usr/lib/python3.7/argparse.py", line 293, in format_help
    help = self._root_section.format_help()
  File "/usr/lib/python3.7/argparse.py", line 224, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.7/argparse.py", line 224, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/usr/lib/python3.7/argparse.py", line 310, in _format_usage
    usage = usage % dict(prog=self._prog)
ValueError: unsupported format character 'p' (0x70) at index 8
```

**Comment by sopelj** [comment]
> I have an issue with the argument parser when trying `wazo-provd-cli --help`:
> 
> ```shell
> Traceback (most recent call last):
>   File "/usr/bin/wazo-provd-cli", line 3, in <module>
>     from wazo_provd_cli import cli
>   File "/usr/lib/python3/dist-packages/wazo_provd_cli/cli.py", line 56, in <module>
>     opts, args = parser.parse_known_args()
>   File "/usr/lib/python3.7/argparse.py", line 1797, in parse_known_args
>     self.error(str(err))
>   File "/usr/lib/python3.7/argparse.py", line 2508, in error
>     self.print_usage(_sys.stderr)
>   File "/usr/lib/python3.7/argparse.py", line 2478, in print_usage
>     self._print_message(self.format_usage(), file)
>   File "/usr/lib/python3.7/argparse.py", line 2444, in format_usage
>     return formatter.format_help()
>   File "/usr/lib/python3.7/argparse.py", line 293, in format_help
>     help = self._root_section.format_help()
>   File "/usr/lib/python3.7/argparse.py", line 224, in format_help
>     item_help = join([func(*args) for func, args in self.items])
>   File "/usr/lib/python3.7/argparse.py", line 224, in <listcomp>
>     item_help = join([func(*args) for func, args in self.items])
>   File "/usr/lib/python3.7/argparse.py", line 310, in _format_usage
>     usage = usage % dict(prog=self._prog)
> ValueError: unsupported format character 'p' (0x70) at index 8
> ```

Wow, of all the options to have forgotten to test... sorry. It's fixed.

**Review by bloom1 — APPROVED** [review]

**Pull request merged**